### ISSUE TITLE
Missing "Open Source" in employee principle E2

### ISSUE THREAD

**Issue by gibello** [mercedes-benz/mercedes-benz-foss-manifesto#9]
In "Employee Principles", E2 rule appears as follows:

```
E2. An engineer shall strive to be active in the Inner Source communities.
```

According to the general tone of the manifesto (and other rules), it is likely "Open Source" was forgotten, for example:

```
E2. An engineer shall strive to be active in the Open and Inner Source communities.
```

**Comment by wgehring** [comment]
Hello @gibello ,
thanks very much for you feedback! :-)

Actually, Open Source wasn't forgotten. But because we regard both Inner Source and Open Source as very important, we have devoted E2 only to Inner Source, and E3 only to Open Source. But basically, E3 is the Open Source counterpart to E2. (We just didn't use the exact words in E3.)
Does that make sense?

**Comment by gibello** [comment]
Hello @wgehrig , fine with that, closes #9 !
